src/lottie.css

```diff
@@ -1,13 +1,13 @@
-[data-lottie],
-[data-lottie] .wp-block-media-text__media {
+.wp-site-blocks [data-lottie],
```

Comment:
I think this means it will only work with block themes. What about `[class*="wp-block-"][data-lottie]` to get `0-2-0` specifity?